scripts/contiv-vpp-bug-report.sh

```diff
@@ -124,10 +124,11 @@ for POD in $PODS; do
     get_vpp_data "sh acl-plugin acl" acls
     get_vpp_data "sh hardware-interfaces" hardware-info
     get_vpp_data "sh errors" errors
-    get_vpp_data "api trace save trace.api" api-trace-save
-    get_vpp_data "api trace custom-dump /tmp/trace.api" api-trace-dump
+#     get_vpp_data "api trace save trace.api" api-trace-save
```

Comment:
Please add a comment above these explaining why they are commented out, ideally with a link to a bug.